tests/unit/Jadu/Pulsar/Twig/Macro/Fixtures/visuals/label-immutability-test.html.twig

```diff
@@ -0,0 +1,271 @@
+            class: 'btn--warning'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'btn--danger'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'btn--info'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'btn--inverse'
+        })
+
+    }}
+</div>
+
+<hr />
+<h3>Test labels inside disabled buttons</h3>
+<div class="btn__toolbar">
+    text {{
+        html.button({
+            'label': html.label({ 'label': '123' }),
+            'class': 'is-disabled'
+        })
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--primary'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--success'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--warning'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--danger'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--info'
+        })
+
+    }}
+    {{
+        html.button({
+            label: html.label({ 'label': '123' }),
+            class: 'is-disabled btn--inverse'
+        })
+
+    }}
+</div>
+<hr />
+<h3>Test labels inside disabled table row</h3>
+
+<div class="btn__toolbar">
+    <table class="table">
+        <tr class="is-disabled">
+            <td>
+                text {{
+                    html.label({
+                        label: '1'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '2', class: 'label--primary'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '3', class: 'label--success'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '4', class: 'label--warning'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '5', class: 'label--danger'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '6', class: 'label--info'
+                    })
+                }}
+                {{
+                    html.label({
+                        label: '7', class: 'label--inverse'
+                    })
+                }}
+            </td>
+        </tr>
+    </table>
+</div>
+{% endblock %}
```

Comment:
Make empty lines consistent throughout file, sometimes they're before the `<hr>`, other times after.